test/uk/gov/hmrc/taxhistory/controllers/PayAsYouEarnControllerSpec.scala

```diff
@@ -17,31 +17,31 @@
 package uk.gov.hmrc.taxhistory.controllers
 
 import org.joda.time.LocalDate
-import org.mockito.Matchers._
+import org.mockito.Matchers.{any, eq => meq}
```

Comment:
use original names?

Reply:
Original name from the Java library is `eq` which conflicts with Scala's completely different `eq`, hence the rename.